frontend/src/metabase/core/components/AccordionList/AccordionList.unit.spec.js

```diff
@@ -100,8 +101,8 @@ describe("AccordionList", () => {
         />,
       );
 
-      userEvent.hover(screen.getByText("Foo"));
-      expect(await screen.findByText("popover")).toBeVisible();
+      await user.hover(screen.getByText("Foo"));
+      await waitFor(() => expect(screen.findByText("popover")));
```

Comment:
This is not a correct check — there is no "popover" dom node, so seems like the test should be fixed